packages/luna/src/components/Lessons/RateLesson/RateLesson.tsx

```diff
@@ -0,0 +1,122 @@
+import { Dialog } from "@/components/Dialog/V2";
+import { Typography } from "@/components/Typography";
+import { useFormatMessage } from "@/hooks/intl";
+import { Void } from "@litespace/types";
+import React, { useCallback, useState } from "react";
+import Rate from "@litespace/assets/Rate";
+import { RatingStars } from "@/components/RatingStars";
+import { Button } from "@/components/Button";
+import { Textarea } from "@/components/Textarea";
+
+export const RateLesson: React.FC<{
+  close: Void;
+  tutorName: string | null;
+  studentId: number;
+  rateLoading: boolean;
+  initialRating?: number;
+  /**
+   * whether you are rating session or the platform itself
+   */
+  type: "session" | "platform";
+  onRate: ({
+    id,
+    value,
+    feedback,
+  }: {
+    /**
+     * ratee id (student)
+     */
+    id: number;
```

Comment:
No need to pass the ratee id to the `onRate` handler. The parent component is already aware of it. This is not the case for feedback and value. 

```suggestion
```